qdrant-landing/content/documentation/Tutorials & Examples/cloud-inference-hybrid-search.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Using Cloud Inference with Qdrant for Vector Search
+weight: 35
+---
+# Using Cloud Inference with Qdrant for Vector Search
+## Install Qdrant Client
+```bash
+pip install qdrant-client
+```
+## Initialize the Client
+Initialize the Qdrant client after creating a [Qdrant Cloud account](/documentation/cloud/) and a [dedicated paid cluster](/documentation/cloud/create-cluster/). 
+```python
+from qdrant_client import QdrantClient
+client = QdrantClient(
+    url="https://YOUR_URL.eastus-0.azure.cloud.qdrant.io:6333/",
+    api_key="YOUR_API_KEY",
+    cloud_inference=True,
+    timeout=30.0
+)  
+```
+## Create a Collection
+Qdrant stores vectors and associated metadata in collections. A collection requires vector parameters to be set during creation. In this case, let's setup a collection using `BM25` for sparse vectors and `all-minilm-l6-v2` for dense vectors. 
+```python
+from qdrant_client import models
+
+collection_name = "my_collection_name"
+if not client.collection_exists(collection_name=collection_name):
+# Create collection with multi-vector config: "bm25_sparse_vector" for BM25, "dense" for MiniLM
+    client.create_collection(
+    collection_name=collection_name,
+    vectors_config={
+        "dense": models.VectorParams(
```

Comment:
Could you please rename the vector to "dense_vector", so the naming is consistent with "bm25_sparse_vector"?